Practice Areas/Engineering/RFC/001-internal-tools-access.md

```diff
@@ -0,0 +1,56 @@
+# Internal Tool Access
+
+The following tools are currently being used by developers and operators for the VSP, currently exposed by the SOCKS proxy. This proxy adds a good deal of toil for operations staff, is not scaleable for onboarding non-VSA engineers to the platform, and is unnecessary now that we have access to VAEC.
+
+Below is a chart with tools we maintain today along with perceived risk profile and exposure tolerance. Risk profile is meant to describe the amount of risk to Veterans and VA if information or access were compromised. Exposure tolerance is meant to describe whether VA might tolerate a service being exposed internally or externally from a compliance perspective.
+
+| Tool | Risk Profile | Exposure Tolerance |
+| --- | --- | --- |
+| Grafana / Prometheus | low | external |
+| Sentry* | high | internal |
+| Jenkins | high** | internal |
+| API gateway management | high | internal |
+| content preview | low | external |
+| review instances | low | external |
+
+*Sentry may need to be switched out for a different technology or moved to hosted solution very soon.
+
+**Jenkins could be split into multiple parts to separate the risk profile, but it's being used as a deployment mechanism right now which raises the risk profile.
+
+## Proposal
+
+While Option 2 outlined below has a higher number of cons, the items in that list are much smaller impact than those in the pro list, and we believe Option 2 should be the path forward.
+
+## Option 1
+
+Expose developer tools on https://dev.va.gov/_vfs/{toolname} ({toolname} here would be like "grafana", "sentry", etc).
```

Comment:
Did you pick `dev.va.gov` because its a subdomain already in place?  I'd expect `dev.va.gov` and `staging.va.gov` to have the same content, just different environments.  What about something like `devtools.va.gov`?

Reply:
Yeah, picked it because we already have the domain provisioned. Getting a new domain exposed is a good deal of work.